w3af/core/controllers/plugins/auth_plugin.py

```diff
@@ -278,10 +279,12 @@ def _log_info_to_kb(self, title, message, include_log_messages=True):
                        '%s')
             desc = msg_fmt % args
 
-        i = Info(title,
+        i = Vuln(title,
                  desc,
+                 severity.INFORMATION,
                  self._http_response_ids,
-                 self.get_name())
+                 self.get_name(),
+                 vulndb_id=10116)
```

Comment:
https://github.com/vulndb/data/search?q=10116&unscoped_q=10116

What happens when the ID is not present in the `vulndb`? Is this error handled somewhere?

Reply:
There's no error. I don't implement here the concept of making `vulndb_id` required for `Vuln` instances. I just mentioned the concept in description, because in this PR I prepare the code for this concept. I'll be able to implement it once https://github.com/andresriancho/w3af/pull/18543 is merged.

Reply:
My point is, when `10116` is passed to the `Vuln` instance, and then XML plugin tries to write that vuln instance to disk, what happens? Will it crash because `10116` is NOT in the `vulndb`?

`10116` is Holm-specific, that is why I'm asking.